pennylane/transforms/__init__.py

```diff
@@ -20,38 +20,32 @@
 Transforms
 ----------
 
-Transforms that act on QNodes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+How to build a custom PennyLane transform?
```

Comment:
Looking at the built page: https://xanaduai-pennylane--4682.com.readthedocs.build/en/4682/code/qml_transforms.html

I think we need this (and the following section headers) to be moved one level lower. "Transforms" has the `-` level.

Reply:
I merged the two titles